SwiftCompilerSources/CMakeLists.txt

```diff
@@ -245,11 +245,8 @@ else()
 #define COMPILED_WITH_SWIFT
 
 #include \"Basic/BasicBridging.h\"
-#include \"AST/ASTBridging.h\"
```

Comment:
Just checked and it looks like Basic/SourceLoc.swift is still importing `ASTBridging`, that should probably be changed to import `BasicBridging`

Reply:
Oooh, thanks!